app/org/sagebionetworks/bridge/stormpath/StormpathAccountDao.java

```diff
@@ -314,21 +312,21 @@ public void deleteAccount(Study study, String email) {
     }
     
     private void rethrowResourceException(ResourceException e, Account account) {
+        logger.info(String.format("Stormpath error: %s, exception: %s, mapped to BadRequestException", e.getCode(), e.getMessage()));
```

Comment:
The exception can be mapped to other exceptions, not just BadRequestException, right?